File: common/src/main/java/com/wdz/common/room/entity/History.java
package com.wdz.common.room.entity;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

/**
 * @Author dezhi.wang
 * @Date 2021/2/1 17:07
 */
@Entity(tableName = "history_tab")
public class History {
    @PrimaryKey(autoGenerate = true)
    @NonNull
    public int id;

    @ColumnInfo(name = "search_info")
    public String searchTitle;

    @Override
    public String toString() {
        return "History{" +
                "id=" + id +
                ", searchTitle='" + searchTitle + '\'' +
                '}';
    }
}

File: kt_common/src/main/java/com/wdz/ktcommon/room/entity/History.kt
package com.wdz.ktcommon.room.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * @Author dezhi.wang
 * @Date 2021/2/1 17:07
 */
@Entity(tableName = "history_tab")
class History {
    @PrimaryKey(autoGenerate = true)
    var id = 0

    @ColumnInfo(name = "search_info")
    var searchTitle: String? = null
    override fun toString(): String {
        return "History{" +
                "id=" + id +
                ", searchTitle='" + searchTitle + '\'' +
                '}'
    }
}